app/i18n/lang/hu.json

```diff
@@ -35,5 +36,6 @@
 		"invalidServerTimeout": "A kiszolgáló időkorlátja érvénytelen",
 		"serverUrl": "Kiszolgáló URL-címe",
 		"serverTimeout": "Kiszolgálói időkorlát (másodperc)"
```

Comment:
Comma missing at the end of the line (as its no longer the last line)

Reply:
Done, thank you